tools/oversight/slicer.js

```diff
@@ -261,6 +263,11 @@ function updateDataFacets(filterText, params, checkpoint) {
       if (cp === 'acquisition') {
         dataChunks.addFacet('acquisition.source', acquisitionSource);
       }
+
+      // special handling for enter checkpoint
+      if (cp === 'enter') {
+        dataChunks.addFacet('enterSource', enterSource);
```

Comment:
```suggestion
        dataChunks.addFacet('enter.source', enterSource);
```

if we do this, then you don't need the changes to the HTML and CSS files.